packages/constants/src/constants.ts

```diff
@@ -49,17 +83,18 @@ export type TAppNotificationOfKind<
     ? TAppNotificationKindSide
     : never;
 };
-export type TAppNotificationApiError = {
-  code: string;
+export type TAppNotificationApiError<ExtraFields extends {} = {}> = {
```

Comment:
The `ExtraFields` is used to extend the error object, which is helpful for the API error notifications